src/settings-ui/Settings.UI/SettingsXAML/Controls/KeyVisual/KeyVisual.xaml

```diff
@@ -23,26 +21,6 @@
             <Setter.Value>
                 <ControlTemplate TargetType="local:KeyVisual">
                     <Grid>
```

Comment:
This-level grid can be removed, it has no value

Reply:
The Grid is needed for the <VisualStateManager.VisualStateGroups>, which is not attachable to <ControlTemplate>. But thanks!